guides/keras_nlp/upload.py

```diff
@@ -0,0 +1,167 @@
+"""
+Title: Model Upload
```

Comment:
`Uploading Models with KerasNLP`?

Just for parallelism with `Getting Stated with KerasNLP`.